src/components/DataMessenger/DataMessenger.js

```diff
@@ -46,6 +47,79 @@ import { runDrilldown, fetchQueryTips } from '../../js/queryService'
 import 'rc-drawer/assets/index.css'
 import './DataMessenger.scss'
 
+let strings = new LocalizedStrings({
```

Comment:
Can we change the word "strings" to something like "lang" or "language" to better represent what we're referring to?

Reply:
changed to lang